### ISSUE TITLE
License files are not included in the snafu-derive package

### ISSUE THREAD

**Issue by tim77** [shepmaster/snafu#215]
Hello. Please add license files to Crates.io for `snafu-derive`. For some Linux distros this is mandatory.

**Comment by shepmaster** [comment]
Can you expand a bit more what you mean? `snafu` and `snafu-derive` are under the same license and appear the same in crates.io:

![image](https://user-images.githubusercontent.com/174509/76963914-2d55d780-68f8-11ea-8ae2-3c07fc97b5bb.png)


![image](https://user-images.githubusercontent.com/174509/76963943-3cd52080-68f8-11ea-98c1-49345d04d556.png)

**Comment by tim77** [comment]
License files available only for `snafu` not `snafu-derive` on Crates.io. Simply creating symlink to the top-level license files from a `snafu` directory would be enough.

**Comment by shepmaster** [comment]
> License files available only for `snafu` not `snafu-derive` on Crates.io

I still am not following you. Where in the crates.io interface is anything shown about licenses beyond the points that I have circled?

**Comment by tim77** [comment]
**snafu-derive**

```sh
snafu-derive-0.6.2 is 📦 v0.6.2 via 🦀 v1.42.0

ls -l
total 12
-rw-r--r--. 1 tim tim 1027 Jan 17 18:44 Cargo.toml
-rw-r--r--. 1 tim tim  464 Jan 17 18:41 Cargo.toml.orig
drwxr-xr-x. 2 tim tim 4096 Mar 18 15:17 src
```

**snafu**

![DeepinScreenshot_select-area_20200318152051](https://user-images.githubusercontent.com/5614476/76964903-1c26be00-692c-11ea-903d-0ea2c55d7db2.png)

**Comment by shepmaster** [comment]
Ah, so you mean the *files* that are bundled as part of the distributed source code. That was not clear.

**Comment by tim77** [comment]
Exactly. :) Apologize, i should clarify this better.

**Comment by ignatenkobrain** [comment]
It is enough to just create symlinks or copy LICENSE files into subfolder snafu-derive, commit them and publish new version.

**Comment by shepmaster** [comment]
Released in 0.6.3. I'm interested to hear which Linux distro might be bundling SNAFU, if you can share.

**Comment by tim77** [comment]
> Released in 0.6.3.

Thank you!

> I'm interested to hear which Linux distro might be bundling SNAFU, if you can share.

Fedora. Packaging Rust software in Fedora is PITA.
